src/tasks/DockerPseudoterminal.ts

```diff
@@ -56,33 +63,52 @@ export class DockerPseudoterminal implements Pseudoterminal {
     public async executeCommandInTerminal(
         command: CommandLineBuilder,
         folder: WorkspaceFolder,
-        rejectOnStderr?: boolean,
-        stdoutBuffer?: Buffer,
-        stderrBuffer?: Buffer,
         token?: CancellationToken): Promise<void> {
         const commandLine = resolveVariables(command.build(), folder);
 
         // Output what we're doing, same style as VSCode does for ShellExecution/ProcessExecution
         this.write(`> ${commandLine} <\r\n\r\n`, DEFAULTBOLD);

```

Comment:
This is being output correctly but is pretty quickly overwritten when the build task starts; it completely clears the terminal window. The same does not happen for shell commands. Should try to fix it.

Reply:
Alex suggested some control character changes that might help.